planner/core/plan_to_pb.go

```diff
@@ -219,6 +219,13 @@ func (p *PhysicalTableScan) ToPB(ctx sessionctx.Context, storeType kv.StoreType)
 	keepOrder := p.KeepOrder
 	tsExec.KeepOrder = &keepOrder
 	tsExec.IsFastScan = &(ctx.GetSessionVars().TiFlashFastScan)
+	// sc := ctx.GetSessionVars().StmtCtx
```

Comment:
If we change pb, can we pass the CI test only by commenting those codes? Can we specify a pb pr to compile and pass CI?

Reply:
No, Bazel will not pull `go replace`, I will change this after the pr in `tipb` merged.